src/components/Header.js

```diff
@@ -4,9 +4,14 @@ import PropTypes from 'prop-types';
 function Header({ subtitle }) {
   return (
     <Container>
-      <div style={{ margin: '25px 0' }}>
-        <Typography variant="h4" color="textSecondary" align="center">
-          {subtitle}
+      <div style={{ margin: '15px 0' }} data-cy="header">
```

Comment:
here, we can use `<Box my={2 or something}`. And alse set the fontWeight here too.